lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -31,6 +31,9 @@ export default function FilePickerApp({
     authToken,
     authUrl,
     courseId,
+    // It is assumed that enableLmsFilePicker is only used by LMS apps
+    // for which we do not have an integration with the file storage.
+    // (currently only Canvas supports this).
```

Comment:
This comment is true at the time of writing, but we definitely do intend to support file storage for other LMSes in future and Jon and Sean have been actively looking into it.